inference-engine/tests/functional/plugin/gpu/shared_tests_instances/behavior/config.cpp

```diff
@@ -99,80 +47,117 @@ namespace {
+                    {InferenceEngine::PluginConfigParams::KEY_CONFIG_FILE, "unknown_file"}},
+            {{InferenceEngine::MultiDeviceConfigParams::KEY_MULTI_DEVICE_PRIORITIES , CommonTestUtils::DEVICE_GPU},
+                    {InferenceEngine::PluginConfigParams::KEY_DUMP_KERNELS, "ON"}},
+            {{InferenceEngine::MultiDeviceConfigParams::KEY_MULTI_DEVICE_PRIORITIES , CommonTestUtils::DEVICE_GPU},
+                    {InferenceEngine::PluginConfigParams::KEY_TUNING_MODE, "TUNING_UNKNOWN_MODE"}},
+            {{InferenceEngine::MultiDeviceConfigParams::KEY_MULTI_DEVICE_PRIORITIES , CommonTestUtils::DEVICE_GPU},
+                    {InferenceEngine::PluginConfigParams::KEY_DEVICE_ID, "DEVICE_UNKNOWN"}}
+    };
 
-    INSTANTIATE_TEST_SUITE_P(smoke_Multi_BehaviorTests, IncorrectConfigAPITests,
-            ::testing::Combine(
-                    ::testing::ValuesIn(netPrecisions),
-                    ::testing::Values(CommonTestUtils::DEVICE_MULTI),
-                    ::testing::ValuesIn(multiinconfigs)),
-            IncorrectConfigAPITests::getTestCaseName);
+    INSTANTIATE_TEST_SUITE_P(smoke_Multi_BehaviorTests, IncorrectConfigTests,
+                             ::testing::Combine(
+                                     ::testing::ValuesIn(netPrecisions),
+                                     ::testing::Values(CommonTestUtils::DEVICE_MULTI),
+                                     ::testing::ValuesIn(incorrectMultiConfigs)),
+                             IncorrectConfigTests::getTestCaseName);
 
-    INSTANTIATE_TEST_SUITE_P(smoke_Auto_BehaviorTests, IncorrectConfigAPITests,
-            ::testing::Combine(
-                    ::testing::ValuesIn(netPrecisions),
-                    ::testing::Values(CommonTestUtils::DEVICE_AUTO),
-                    ::testing::ValuesIn(autoinconfigs)),
-            IncorrectConfigAPITests::getTestCaseName);
+    INSTANTIATE_TEST_SUITE_P(smoke_Multi_BehaviorTests, IncorrectConfigAPITests,
+                             ::testing::Combine(
+                                     ::testing::ValuesIn(netPrecisions),
+                                     ::testing::Values(CommonTestUtils::DEVICE_MULTI),
+                                     ::testing::ValuesIn(incorrectMultiConfigs)),
+                             IncorrectConfigAPITests::getTestCaseName);
 
-    INSTANTIATE_TEST_SUITE_P(smoke_AutoCG_BehaviorTests, IncorrectConfigAPITests,
+    INSTANTIATE_TEST_SUITE_P(smoke_BehaviorTests, IncorrectConfigAPITests,
                             ::testing::Combine(
-                                ::testing::ValuesIn(netPrecisions),
-                                ::testing::Values(CommonTestUtils::DEVICE_AUTO),
-                                ::testing::ValuesIn(autoinconfigs)),
+                                    ::testing::ValuesIn(netPrecisions),
+                                    ::testing::Values(CommonTestUtils::DEVICE_GPU),
+                                    ::testing::ValuesIn(config)),
```

Comment:
IncorrectConfig tests better use inconfig, also I think it is not necessary to change the original tests.